website/docs/patterns/form-patterns/partials/guidelines/guidelines.md

```diff
@@ -247,3 +247,15 @@ For complex forms, indicate **required** fields. This is the most explicit and t
 For shorter, simpler forms, indicate **optional** fields instead.
 
 Don't mix required and optional labels, stick to one or the other. Using one method can imply that fields without a label are the inverse of whatever method you choose, though expecting the user to carry this knowledge through a complex form can cause unnecessary [validation](?tab=validation) errors when required fields aren't labeled.
+
+## Composition with other components
+
+While form patterns should generally be comprised of form-specific components, sometimes it may be necessary to compose categorically different components together to satisfy a user need.
```

Comment:
I'm not sure this content and heading are needed, given that pattern documentation is always about composing various components to make a larger feature.